src/expr/dtype.cpp

```diff
@@ -897,8 +897,6 @@ const std::vector<std::shared_ptr<DTypeConstructor> >& DType::getConstructors()
 
 std::ostream& operator<<(std::ostream& os, const DType& dt)
 {
-  // can only output datatypes in the cvc5 native language
-  language::SetLanguage::Scope ls(os, Language::LANG_CVC);
```

Comment:
I assume the comment was incorrect already?

Reply:
We can print datatypes in SMT-LIBv2. Not sure when this comment got obsolete. @ajreynol?

Reply:
This comment is extremely obsolete.